app/experimenter/experiments/tests/test_models.py

```diff
@@ -2143,6 +2156,25 @@ def test_latest_rejection_returns_rejection_for_review_to_idle(self):
         self.assertEqual(experiment.changes.latest_review_request(), changes[0])
         self.assertEqual(experiment.changes.latest_rejection(), changes[1])
 
+    def test_latest_rejection_returns_rejection_for_review_to_dirty(self):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LIVE_DIRTY,
```

Comment:
In the above test, you have included `is_rollout=True`, but not here, If its included in life cycle then its good and would recommend to remove from the above test too, if not I will suggest to add it here too

Reply:
Ah! great catch!! It should be added here too